crates/sui-framework/packages/move-stdlib/sources/fixed_point32.move

```diff
@@ -90,20 +107,63 @@ module std::fixed_point32 {
         FixedPoint32 { value: quotient as u64 }
     }
 
+    /// Create a fixed-point value from an integer.
+    public fun from_integer(integer: u32): FixedPoint32 {
+        from_raw((integer as u64) << 32)
+    }
+
     /// Create a fixedpoint value from a raw value.
-    public fun create_from_raw_value(value: u64): FixedPoint32 {
+    public fun from_raw(value: u64): FixedPoint32 {
```

Comment:
I think the names are good. But we cannot rename them this way. We have to 
1. Make a new function
2. Deprecate the old function